File: pubnub-gson/pubnub-gson-api/src/main/java/com/pubnub/api/java/models/consumer/datasync/user/PNRemoveUserResult.java
package com.pubnub.api.java.models.consumer.datasync.user;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Getter
@NoArgsConstructor
@ToString
public class PNRemoveUserResult {
    private int status;

    public PNRemoveUserResult(int status) {
        this.status = status;
    }
}

File: pubnub-entities/src/main/kotlin/com/pubnub/entities/models/consumer/user/PNRemoveUserResult.kt
package com.pubnub.entities.models.consumer.user

import com.pubnub.api.models.consumer.objects.PNRemoveMetadataResult

data class PNRemoveUserResult(
    val status: Int
)

internal fun PNRemoveMetadataResult.toPNRemoveUserResult(): PNRemoveUserResult {
    return this?.let { PNRemoveUserResult(status = it.status) }
}